### PR TITLE
handle no scopes

### PR DESCRIPTION
**Pull request by jennyf19**
#14

### CODE REVIEW

File: tests/B2CWebAppCallsWebApi/Client/Startup.cs [hunk 1]
```diff
@@ -53,9 +53,7 @@ public void ConfigureServices(IServiceCollection services)
 
             // Token acquisition service based on MSAL.NET
             // and chosen token cache implementation
-            services.AddWebAppCallsProtectedWebApi(Configuration,
-                initialScopes: new string[] { Configuration["TodoList:TodoListScope"] },
-                configSectionName: "AzureAdB2C")
+            services.AddWebAppCallsProtectedWebApi(Configuration, "AzureAdB2C")
```
Comment (bgavrilMS) [inline tests/B2CWebAppCallsWebApi/Client/Startup.cs:56]: Just wanted to point out the difference between AAD and B2C: 

- AAD will return an access token if you pass 0 scopes. AcquireTokenSilent works.
- B2C will not return an access token if you pass 0 scopes. This is a known issue that B2C team plans to fix at some point. AcquireTokenSilent does not work (MSAL will thrown an exception). AcquireToken*** will still return an ID token though.

So, long story short, I'm not sure you want to make this change for B2C ?

Comment (jennyf19) [reply tests/B2CWebAppCallsWebApi/Client/Startup.cs:56]: yes. i just talked about it w/ @jmprieur .I think we will document it for now. I'll let @jmprieur make that decision.

File: tests/B2CWebAppCallsWebApi/Client/Startup.cs [hunk 2]
```diff
@@ -53,7 +53,9 @@ public void ConfigureServices(IServiceCollection services)
 
             // Token acquisition service based on MSAL.NET
             // and chosen token cache implementation
-            services.AddWebAppCallsProtectedWebApi(Configuration, "AzureAdB2C")
+            services.AddWebAppCallsProtectedWebApi(Configuration,
+                     initialScopes: new string[] { Configuration["TodoList:TodoListScope"] },
+                     configSectionName: "AzureAd")
```
Comment (jmprieur) [inline tests/B2CWebAppCallsWebApi/Client/Startup.cs:58]:
```suggestion
                     configSectionName: "AzureAdB2C")
```

File: src/Microsoft.Identity.Web/WebAppServiceCollectionExtensions.cs [hunk 3]
```diff
@@ -90,6 +90,31 @@ public static class WebAppServiceCollectionExtensions
...
+        /// Add MSAL support to the Web App or Web API.
+        /// </summary>
+        /// <param name="services">Service collection to which to add authentication.</param>
+        /// <param name="configuration">Configuration.</param>
+        /// <param name="configSectionName">The name of the configuration section with the necessary
+        /// settings to initialize authentication options.</param>
+        /// <param name="openIdConnectScheme">Optional name for the open id connect authentication scheme
+        /// (by default OpenIdConnectDefaults.AuthenticationScheme). This can be specified when you want to support
+        /// several OpenIdConnect identity providers.</param>
+        /// <returns>The service collection for chaining.</returns>
```
Comment (jmprieur) [inline src/Microsoft.Identity.Web/WebAppServiceCollectionExtensions.cs:103]:
```suggestion
        /// <returns>The service collection for chaining.</returns>
        /// <remarks>This method cannot be used with Azure AD B2C as, with B2C an initial scope needs
        /// to be provided</remarks>
```

**Pull request merged**